sweep/fee_bumper.go

```diff
@@ -246,10 +253,20 @@ type BumpResult struct {
 	requestID uint64
 }
 
+// String returns a human-readable string for the result.
+func (b *BumpResult) String() string {
+	desc := fmt.Sprintf("Event=%v", b.Event)
+	if b.Tx != nil {
+		desc += fmt.Sprintf(", Tx=%v", b.Tx.TxHash())
+	}
+
+	return fmt.Sprintf("[%s]", desc)
+}
+
 // Validate validates the BumpResult so it's safe to use.
 func (b *BumpResult) Validate() error {
-	// Every result must have a tx.
-	if b.Tx == nil {
+	// Every result must have a tx except the error or fail case.
+	if b.Tx == nil && b.Event != TxFatal {
```

Comment:
just checking fatal case in this commit but comment says error or fail case

Reply:
ah ok i see it is fixed in upcoming commit - so nbd. but maybe "failed or fatal" makes more sense in the comment

Reply:
fixed the comment